code/drasil-example/projectile/app/Main.hs

```diff
@@ -12,11 +14,14 @@ import qualified Drasil.Projectile.Lesson.Body as PL (nb, printSetting)
 main :: IO()
 main = do
   setLocaleEncoding utf8
+  dumpEverything fullSI ".drasil/"
   typeCheckSI fullSI
   gen (DocSpec (docChoices SRS [HTML, TeX, JSON]) "Projectile_SRS") srs printSetting
   gen (DocSpec (docChoices Jupyter [])      "Projectile Motion Lesson") PL.nb PL.printSetting
   genCodeWithChoices choiceCombos
   genDot fullSI
   genLog fullSI printSetting
+  -- if the chunkDB had a mutable state, then this would make more sense.
+  -- dumpChunkDB fullSI ".drasil/" "fruits.json"
```

Comment:
Not sure what this means lol

Reply:
I think of the user-inputted information as the `seeds`, the Drasil-provided features as the `soil`, and the final tree of things automatically generated from the `seeds` as the `fruits`. So, if we had more gradual generation, we can take snapshots (breakpoints) of the `ChunkDB` at specific spots during generation.

Reply:
Nice!